js/battle.js

```diff
@@ -524,18 +524,21 @@ var Pokemon = (function () {
 			this.baseAbility = ability;
 		}
 	};
+	Pokemon.prototype.prepName = function (name) {
```

Comment:
I guess that's fair. That makes me wonder I should decide what to reconstruct the nickname as, `getName`, `getLowerName`, or `getFullName`?